src/analysis/processing/qgsalgorithmfilter.cpp

```diff
@@ -92,6 +96,8 @@ QVariantMap QgsFilterAlgorithm::processAlgorithm( const QVariantMap &parameters,
   for ( Output *output : qgis::as_const( mOutputs ) )
   {
     output->sink.reset( parameterAsSink( parameters, output->name, context, output->destinationIdentifier, source->fields(), source->wkbType(), source->sourceCrs() ) );
+    if ( !output->sink )
```

Comment:
I think this should throw a QgsProcessingException instead - it's a fatal error, we don't want to silently ignore it and continue with the model execution.

Reply:
That was just copied from the buffer algorithm, I also found that a bit strange.

Reply:
I'll fix that

Reply:
Cool, thanks. The one for here is in the pipeline too

Reply:
https://github.com/qgis/QGIS/commit/015665688262f95e148972d699fc2270d622a75c